### ISSUE TITLE
building a working application errors

### ISSUE THREAD

**Issue by lewislepton** [HaxeIDE/HIDE#210]
just got HIDE to work. thank you @as3boyan 
but on runinng a working OPENFL program, it errors.
i have checked the settings, but nothing is actually changing. i have changed the haxelib to openfl in the settings. but nothing is working

thanks

```
Build started
haxelib run lime build "/Users/lepton/Developer/HAXE/GAMES/BROLLY" neko -debug --connect 5000
stderr:
[31;1mError:[0m [1mYou must have a "project.xml" file or specify another valid project file when using the 'build' command[0m
Build failed (exit code: 1)
```

**Comment by lewislepton** [comment]
and these are the project settings as well

![screen shot 2015-05-30 at 17 46 44](https://cloud.githubusercontent.com/assets/11299322/7898304/e0454216-06f3-11e5-8971-5b4ecb5b39a1.png)

**Comment by lewislepton** [comment]
or i should say the run command does revert right back to 'lime'

**Comment by lewislepton** [comment]
even a clean project does not work with those same settings. very strange

**Comment by lewislepton** [comment]
do you know what may be causing this problem, because i still have not been able to get anytihng to work yet

thanks

**Comment by lewislepton** [comment]
hmmm. still having the same trouble. if you know what could be wrong?

**Comment by vroad** [comment]
I have the same issue. I'm wondering why since building from command line just works fine.

**Comment by daPhyre** [comment]
I confirm I have this problem too. My solution for now was to build a batch file with the command to compile, and double click on it to test the project, but I would certainly love more if I could try so without getting out of the IDE...

**Comment by lewislepton** [comment]
ive kind of left it now to be honest. because its not going to get updated anymore from what i know. since @as3boyan is now with the jetbrains crew for that plugin

im currently using peppermint editor now since that is being actively looked at. the sublime option is also good

**Comment by lewislepton** [comment]
but also as well. a version of flashdevelop is also being looked at just for haxe. called haxedevelop. it will be time till something is concrete, but that would be a future option. plus it should be getting multiplatform built as well. though its just windows for the time being

**Comment by vroad** [comment]
Removing double quotions from project file path("%path%") seems to solve the issue.

**Comment by lewislepton** [comment]
@vroad would that be in HIDE or within the project.xml?
can you screenshot it please ;)

**Comment by as3boyan** [comment]
Sorry guys, sorry for late reply, it seems like it is broken, I think I found another issue:

I tried to create a OpenFL - project, it seems like it creates project, but HIDE can't automatically open newly created project for some reason, so I think I will start from here

@lewislepton You right, I work on Haxe plugin for IntelliJ IDEA and stuff with TiVo

**Comment by lewislepton** [comment]
@as3boyan 
just to ask. will you be updating or in some way finishing off HIDE. its a lovely project, and would be sad to see it dwindle away

thanks for getting back though ;)

**Comment by vroad** [comment]
@lewislepton I've also tried setting custom build command but didn't work.
Modifying source code of HIDE did the trick.

**Comment by as3boyan** [comment]
Sadly HIDE misses lot of stuff like

1. Find in Files
1. Git integration (perhaps there are some JS projects which can be used to achieve this, ungit maybe)
1. Also it misses unit tests

I had to make fixes to get it to compile with Haxe 3.2
https://github.com/HaxeIDE/HIDE/commit/af8275213151c104caf9c5b88902ec0121ee5332

Now it needs a fix to get it to run, seems like it misses something minor, like `nodejs = {}` object to assign other variables or other thing not sure, need to look.

About work on HIDE - maybe I would try to get it to work, but still need to work on IDEA stuff etc and do my daily work.

**Comment by lewislepton** [comment]
@as3boyan 
just to say thanks for putting the time back into this.
ive actually tried jetbrains with haxe and it was nice, though there is a major bug for osx that pretty much buggered it all up for me. its being looked at.

but thanks for putting the time into this again. i know you are busy and have other irons in the fire to deal with
